src/Breadcrumb.js

```diff
@@ -1,27 +1,61 @@
 import classNames from 'classnames';
 import React from 'react';
+import PropTypes from 'prop-types';
 
 import BreadcrumbItem from './BreadcrumbItem';
 import { bsClass, getClassSet, splitBsProps } from './utils/bootstrapUtils';
 
+const propTypes = {
+  /**
+   * ARIA label for the nav element
+   * https://www.w3.org/TR/wai-aria-practices/#breadcrumb
+   */
+  label: PropTypes.string,
+  /**
+   * Additional props passed as-is to the underlying `<ul>` element
+   */
+  listProps: PropTypes.object
+};
+
+const defaultProps = {
+  label: 'breadcrumbs',
+  listProps: {}
+};
+
 class Breadcrumb extends React.Component {
   render() {
-    const { className, ...props } = this.props;
+    const {
+      bsClass,
+      listProps,
+      children,
+      className,
+      label,
+      ...props
+    } = this.props;
     const [bsProps, elementProps] = splitBsProps(props);
 
     const classes = getClassSet(bsProps);
 
     return (
-      <ol
-        {...elementProps}
-        role="navigation"
-        aria-label="breadcrumbs"
+      <nav
+        aria-label={label}
         className={classNames(className, classes)}
-      />
+        {...elementProps}
+      >
+        <ol
+          role="navigation"
+          className={classNames(bsClass, listProps.className)}
+          {...listProps}
```

Comment:
switch the order here to:

```
role="navigation"
{...listProps}
className={classNames(bsClass, listProps.className)}
```

otherwise it'll still override the className 👍